library/general/requiredannotations/template.yaml

```diff
@@ -18,13 +19,20 @@ spec:
               type: string
             annotations:
               type: array
+              description: >-
+                A list of annotations and values the object must specify.
               items:
                 type: object
                 properties:
                   key:
                     type: string
+                    description: >-
+                      The required annotation.
                   allowedRegex:
                     type: string
+                    description: >-
+                      A regular expression the annotation's value must match. The value
```

Comment:
We should probably mention that this field is optional, and that the regex wont be checked if `allowedRegex` is undefined.

Reply:
Done!